src/debug/debug-pal/unix/diagnosticsipc.cpp

```diff
@@ -61,6 +60,8 @@ IpcStream::DiagnosticsIpc *IpcStream::DiagnosticsIpc::Create(const char *const p
         pd.m_ApplicationGroupId,
         "socket");
 
+    ::unlink(serverAddress.sun_path);
```

Comment:
if I remember correctly, doesn't `unlink` return something on failure/success? Maybe we should add a retry logic here based on what this returns.

Reply:
Yes, it does return success or failure [unlink(2)](https://linux.die.net/man/2/unlink).
Here I'm trying to remove an already existing `socket`. This is the case when I'm debugging/testing the feature, I have hardcoded the IDs, and I do not exit the runtime properly (Ctrl+C).
I think I should remove this.